crates/node/src/event_source/event_source_actions.rs

```diff
@@ -3,6 +3,7 @@ use serde::{Deserialize, Serialize};
 pub type EventSourceActionWithMeta = redux::ActionWithMeta<EventSourceAction>;
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
+#[expect(clippy::large_enum_variant, reason = "This enum extends super::Event")]
```

Comment:
🤔 I'm not sure if I agree with this one. This should almost certainly be boxed, no?

Also... a nit... but... I would say "wraps," not "extends"